grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json

```diff
@@ -3332,7 +3332,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "${DS_PROMETHEUS}",
-      "description": "Number of mutations (ALTER DELETE/ALTER UPDATE)",
+      "description": "Number of mutations (ALTER DELETE/ALTER UPDATE) which not complete and how much parts should be mutate",
```

Comment:
Please name it 'Number of active mutations (ALTER TABLE UPDATE/DELETE)'.

Reply:
fixed in https://github.com/Altinity/infra/commit/59e09a75f9bc42ee21987aec61d0d0af8bc5feaf